pkg/handler/telegram.go

```diff
@@ -51,9 +54,14 @@ func (h *Handler) LinkUserTelegramWithDiscord(c *gin.Context) {
 	var req request.LinkUserTelegramWithDiscordRequest
 	if err := c.Bind(&req); err != nil {
 		h.log.Error(err, "[handler.LinkUserTelegramWithDiscord] Bind() failed")
-		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, err, nil))
 		return
 	}
+	if req.DiscordID == "" || req.TelegramUsername == "" {
```

Comment:
for post request, can edit `request.LinkUserTelegramWithDiscordRequest` to
```
type LinkUserTelegramWithDiscordRequest struct {
	TelegramUsername string `json:"telegram_username" form:"guild_id" binding:"required"`
	DiscordID        string `json:"discord_id" form:"guild_id" binding:"required"`
}
```
for validating and no need catch `req.DiscordID == ""`